tests/ChatTests.hs

```diff
@@ -107,9 +108,10 @@ chatTests = do
     it "accept contact request incognito" testAcceptContactRequestIncognito
     it "join group incognito" testJoinGroupIncognito
     it "can't invite contact to whom user connected incognito to a group" testCantInviteContactIncognito
-  describe "contact aliases" $ do
+  describe "contact aliases and prefs" $ do
     it "set contact alias" testSetAlias
     it "set connection alias" testSetConnectionAlias
+    it "set contact prefs" testSetContactPrefs
```

Comment:
following on latest changes - we should test changing preferences for incognito contact. Probably have both regular and incognito contact in the same test, and test changing profile and changing global preferences - both should affect regular contact but incognito contact shouldn't receive updates

Reply:
btw there doesn't seem to be an api for changing global preferences, or am I missing it?

Reply:
You mean user's global preferences? Then `/_profile` does this job. And `/_group_profile` for groups

Reply:
> following on latest changes - we should test changing preferences for incognito contact. Probably have both regular and incognito contact in the same test, and test changing profile and changing global preferences - both should affect regular contact but incognito contact shouldn't receive updates

https://github.com/simplex-chat/simplex-chat/pull/1280